Esignet/scripts/Esignet_Test_Script.jmx

```diff
@@ -9402,6 +9467,735 @@ log.info(&quot;Password: &quot; + password)
           <hashTree/>
         </hashTree>
       </hashTree>
+      <ThreadGroup guiclass="ThreadGroupGui" testclass="ThreadGroup" testname="Sign Up Service - Registration Status Complete Flow (Execution)" enabled="true">
```

Comment:
Why is this called as "Sign Up Service - Registration Status Complete Flow (Execution)", why not Sign Up Service - Registration Complete Flow (Execution)

Reply:
Also this was not mentioned in the description of the PR

Reply:
Have added it in the description.

And the thread group name is because in the flow at the end we are checking the status of the registration and also on the confluence page the scenario name is provided as Check registration status. So that is why have kept this name.

Reply:
But registration status is one API, but the process is around the end to end process of registration right ?

Reply:
Yes @vishwa-vyom , the process is for end to end registration. Will update the thread group name.

Reply:
With the latest commit, have updated the thread group name to Sign Up Service - Registration Complete Flow (Execution).